consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundChangeManager.java

```diff
@@ -97,7 +97,7 @@ public RoundChangeManager(
       final SignedData<RoundChangePayload> msg) {
 
     if (!isMessageValid(msg)) {
-      LOG.info("RoundChange message was invalid.");
+      LOG.error("RoundChange message was invalid.");
```

Comment:
probably not an error ... just can't use the message (typically pantheon treats malformed packets as bad, but not terrible, as there are so many actors out there ... its kinda normal)

Reply:
if its normal i'll make it info level.